code/modules/power/singularity/collector.dm

```diff
@@ -131,7 +132,7 @@
 		to_chat(user, "<span class='danger'>Access denied.</span>")
 
 /obj/machinery/power/rad_collector/AltClick(mob/user)
-	if(!user.canUseTopic(src, !issilicon(user)))
+	if(!user.canUseTopic(src, BE_CLOSE))
```

Comment:
Silicons should still not be able to alt click it to toggle it even if they are having it up their ass.